### DISCUSSION TITLE
exist any way to customize devtool layout

### DISCUSSION THREAD

**Discussion by wnunezc** [cefsharp/CefSharp#3386] [General]
hello exist any way to customize devtool layout:

https://stackoverflow.com/a/35560835/4717133

```
ChromiumWebBrowser browser = new ChromiumWebBrowser();
browser.ShowDevTools(); // Opens Chrome Developer tools window
```

output:

![image](https://user-images.githubusercontent.com/1207306/106623527-b262d300-6542-11eb-8654-d14a032f9230.png)

expected:

![image](https://user-images.githubusercontent.com/1207306/106623582-c4dd0c80-6542-11eb-8a3d-d7dc76ea4849.png)

**Comment by amaitland** [comment]
You can show DevTools as a popup or use a parent control to host it as a child within another window.

**Reply by amaitland** [reply]
You can also enable remote debugging and use Chrome. 

Those are the only options CEF supports.
